pipelines/common/src/test/java/com/google/fhir/analytics/view/ViewSchemaTest.java

```diff
@@ -62,19 +62,6 @@ public class ViewSchemaTest {
         ]
       }""";
 
-  private ViewDefinition loadDefinition(String viewFile) throws IOException {
```

Comment:
Any particular reason fro dropping this method?

Reply:
I changed this after removing exception handling in this test class. I think this extra method for View Definition creation was unnecessary.

Reply:
Ok thanks, I decided to go with the old way.